test/core/filters/filter_fuzzer.cc

```diff
@@ -258,40 +241,36 @@ ChannelArgs LoadChannelArgs(const FuzzerChannelArgs& fuzz_args,
 
 #define MAKE_FILTER(name) Filter::Make<name>(#name)
```

Comment:
I think this macro is not needed anymore

Reply:
fixed